Gemfile.lock

```diff
@@ -64,19 +81,19 @@ GEM
     rubocop-ast (1.37.0)
       parser (>= 3.3.1.0)
     ruby-progressbar (1.13.0)
-    temporalio (0.4.0)
+    temporalio (0.5.0)
```

Comment:
Also done in https://github.com/temporalio/samples-ruby/pull/41. Though I see now I may need to go update that to remove the `macos-13` runner